Dockerfile.dev

```diff
@@ -0,0 +1,22 @@
+FROM elixir:1.13-alpine
```

Comment:
I don't see any value in adding a Dockerfile for development. There's nothing to set up, and it makes much harder to work with mix.